TST: test failures for polyons - precision

https://github.com/corteva/gars-field/runs/5824222884?check_suite_focus=true
```
FAILED test/test_field.py::test_gars_1min_exists_for_point - AssertionError: ...
FAILED test/test_gars.py::test_gars_polygon[720HN2604-POLYGON ((179.9833333333333 0.4, 179.9833333333333 0.4166666666666667, 179.9666666666667 0.4166666666666667, 179.9666666666667 0.4, 179.9833333333333 0.4))]
FAILED test/test_gars.py::test_gars_polygon[720HN26-POLYGON ((180 0.3333333333333333, 180 0.4166666666666666, 179.9166666666667 0.4166666666666666, 179.9166666666667 0.3333333333333333, 180 0.3333333333333333))]
```

--- a/test/test_field.py
+++ b/test/test_field.py
@@ -1054,10 +1054,13 @@ def test_gars_1min_exists_for_point():
         # repeat to check cache is the same
         assert len(garsf.gars_1min) == 1
         assert str(garsf.gars_1min[0]) == "173MA4722"
-        assert str(garsf.gars_1min[0].polygon) == (
-            "POLYGON ((-93.71666666666667 42,"
-            " -93.71666666666667 42.01666666666667,"
-            " -93.73333333333333 42.01666666666667,"
-            " -93.73333333333333 42,"
-            " -93.71666666666667 42))"
+        assert garsf.gars_1min[0].polygon.equals_exact(
+            shapely.wkt.loads(
+                "POLYGON ((-93.71666666666667 42,"
+                " -93.71666666666667 42.01666666666667,"
+                " -93.73333333333333 42.01666666666667,"
+                " -93.73333333333333 42,"
+                " -93.71666666666667 42))"
+            ),
+            tolerance=0.5 * 10**-12,
         )

--- a/test/test_gars.py
+++ b/test/test_gars.py
@@ -1,4 +1,5 @@
 import pytest
+import shapely.wkt
 
 from gars_field import GARSGrid
 
@@ -67,7 +68,9 @@ def test_gars_from_latlon(lat, lon, resolution, expected_id):
     ],
 )
 def test_gars_polygon(gars_id, expected_polygon):
-    assert str(GARSGrid(gars_id).polygon) == expected_polygon
+    assert GARSGrid(gars_id).polygon.equals_exact(
+        shapely.wkt.loads(expected_polygon), tolerance=0.5 * 10**-12
+    )
 
 
 @pytest.mark.parametrize(

